package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "leader-board",
+  "version": "1.0.0",
+  "description": "This is an awesome Leader Board",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "gh-pages -d dist",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Please, to run the project locally, add the instruction to run `npm start` or `npm run start`, in this way I could be able to run the project locally on my computer. 🐶